Xamarin.PropertyEditing/ViewModels/BrushPropertyViewModel.cs

```diff
@@ -54,5 +72,13 @@ protected override void OnValueChanged ()
 			base.OnValueChanged ();
 			OnPropertyChanged (nameof (Opacity));
 		}
+
+		protected override void OnPropertyChanged ([CallerMemberName] string propertyName = null)
+		{
+			base.OnPropertyChanged (propertyName);
+			if (propertyName == nameof (PropertyViewModel.ResourceProvider)) {
+				ResourceSelector = (ResourceProvider != null) ? new ResourceSelectorViewModel (ResourceProvider, Editors.Select (ed => ed.Target), Property) : null;
```

Comment:
I'm going to file an issue afterwards for this, but `Editors` is fluid, this would have to be updated when the editors change.